pkgs/development/tools/build-managers/redo-c/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "redo-c";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "leahneukirchen";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "11wc2sgw1ssdm83cjdc6ndnp1bv5mzhbw7njw47mk7ri1ic1x51b";
+  };
+
+  postPatch = ''
```

How about integrating build and installation like this:
```suggestion
  buildPhase = ''
    gcc -o redo redo.c
    sh links.do
  '';
  
  installPhase = ''
    	mkdir -p "$(out)/bin"
￼	    cp --no-dereference redo redo-* "$(out)/bin"
  '';
```
This is more compact and we do not need to add a separate file.

---

Setting the stdenv phases will break the override capability of mkDerivation. Keeping the commands in the Makefile fits the common way building a C package best.